### ISSUE TITLE
Accessibility: Take into account photosensitive mode for dynamic token ring animations

### ISSUE THREAD

**Issue by Drental** [foundryvtt/foundryvtt#11241]
### User Experience

I just noticed that the token ring flashing might get extreme in some cases where multiple tokens are affected by a new status from an aura or similar cases. To make sure that this does not cause undue distress to people with e.g. epilepsy or who otherwise prefer less motion there should be a way to reduce/remove these flashing colors.

You might default such a setting to the prefers-reduced-motion css setting to help people who have accessibility issues with flashing colors.
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

Or it could be hooked into the Photosensitivity mode.

I don't know what the best way to implement this would be, but I will gladly comment on any solution you bring forward.

**Comment by FloRad** [comment]
A reduced motion setting could be used by other things as well, such a token movement, light source animations, etc.
